tests/test_user.py

```diff
@@ -1,6 +1,22 @@
 """Tests for User related functions"""
-from .mockup import Mockup, clean_user
+from .mockup import Mockup
 from tik_manager4.objects import user
+import pytest
+
+
+@pytest.fixture(autouse=True, scope='function')
+def clean_user():
+    # NOTE: There are other modules where clean_user is used as the decorator
+    # under tests/mockup.py.  Since we don't wanna replicate this in each of
+    # these files, this piece of code can be moved to a `conftest.py` (see
+    # pytest documentation) and autouse may be disabled.  In this case, the
+    # clean_user fixture should be explicitly added to all test arguments.
+    # Explicit is better than implicit anyways...
+    m = Mockup()
```

Comment:
These note has a valid point. Do you think maybe we should include this update in this PR?

Reply:
I think it'd be better to tackle it on a separate ticket, as this PR is mainly to introduce the usage.